examples/FieldHooks/ComputeFieldHooks/removeExtraSpaces.ts

```diff
@@ -0,0 +1,9 @@
+import { TextField, Sheet } from '@flatfile/configure'
+
+const removeExtraSpaces = new Sheet('removeExtraSpaces', {
+  departmentName: TextField({
+    compute: (value: any) => {
+      return value.replace(/\s{2,}/g, ' ')
+    },
+  }),
+})
```

Comment:
I would like to see tests that show/verify what happens for the following string.  Reason about "is this what you really want to happen"

`"    asdf"`
`"asdf    "`
`"    as      df"`  - does this replace only one sequence of spaces?
`"Paddy Mullen"`  - does this remove the single character